presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -168,6 +169,42 @@ public void testMySqlCreatedParameterizedVarchar()
         assertUpdate("DROP TABLE mysql_test_parameterized_varchar");
     }
 
+    @Test
+    public void testMySqlTinyInt1Describe()
+            throws Exception
+    {
+        execute("CREATE TABLE tpch.mysql_test_tinyint1 (" +
```

Comment:
Tests run multithreaded, so if we were to keep these as two separate test methods, they would need to use a unique table name. Otherwise you get random failures because they both try to create/drop the same table at the same time (and actually happens frequently when someone forgets this rule).